### PR TITLE
Update project metadata

### PR DESCRIPTION
**Pull request by dimaqq**
See https://test.pypi.org/project/aapns/ (except wonky version)

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,30 +1,36 @@
...
-        'structlog',
-    ],
-    extras_require={
-        'cli': ['click']
-    },
-    entry_points={
-        'console_scripts': [
-            'aapns = aapns.cli:main'
-        ]
+    version="1.0.1",
```
Comment (ojii) [inline setup.py]: we never even had a 1.0.0!

Comment (dimaqq) [reply setup.py]: Oh. my.
Well, what would you rather have, "1.0.0" or "19.1" or "1.0.1"?

Comment (ojii) [reply setup.py]: calver sounds good to me

Comment (dimaqq) [reply setup.py]: Done :)

**Pull request merged**